src/infra/docs/dev-desktop.md

```diff
@@ -70,6 +70,19 @@ git config --global user.name "Your name"
 git config --global user.email "your-email"
 ```
 
+## How to customize your shell
+
+You can configure your default shell by opening a pull request against the
+[Simpleinfra ansible] to modify the `vars_user_config` as shown bloe:
+
+```yaml
+vars_user_config:
+  - username: gh-jdno
+    shell: /usr/bin/zsh
+  - username: gh-WaffleLapkin
+    shell: /usr/bin/fish
+```
```

Comment:
I would probably rephrase this slightly, since I'm not sure every user will know what `simpleinfra` or `ansible` are. Something along the lines of the following maybe?

````suggestion
You can set your default shell on the dev desktops by adding yourself to a
configuration file in the [`rust-lang/simpleinfra`][simpleinfra] repository.
Open `ansible/roles/dev-desktop/defaults/main.yml`, look for the variable
`vars_user_config`, and add yourself to the list.

```yaml
vars_user_config:
  - username: gh-jdno
    shell: /usr/bin/zsh
  - username: gh-WaffleLapkin
    shell: /usr/bin/fish
```

Open a pull request and request a review from `@rust-lang/infra` (or ping us in
`#t-infra` on Zulip).

After the pull request is merged, an infrastructure admin has to deploy the
new configuration to the dev desktops. Only after that will your default shell
be changed.
````

The `[simpleinfra]` link needs to be added, of course, and we can make `ansible/roles/dev-desktop/defaults/main.yml` a link if we want to. Might be nice to make it easier for folks to find the file, although I'm always concerned that we reorganize our code and break a tone of links elsewhere. 🙈